src/test/java/unstable/MenuControlTest.java

```diff
@@ -22,10 +25,17 @@
     @Test
     public void menuControlTest() {
         
+        UI ui = TestController.getUI();
+        PanelControl panelControl = ui.getPanelControl();
+        Preferences testPref = UI.prefs;
+        
+        int vMajor = ui.VERSION_MAJOR;
+        int vMinor = ui.VERSION_MINOR;
+        int vPatch = ui.VERSION_PATCH;
```

Comment:
These can be accessed through `UI`. We shouldn't access static fields through instances. Also these extra variables aren't really needed I think.